subxt/tests/integration/codegen/mod.rs

```diff
@@ -20,7 +20,7 @@
 /// Generate by:
 ///
 /// - run `polkadot --dev --tmp` node locally
-/// - `cargo run --release -p subxt-cli -- codegen | rustfmt --edition=2018 --emit=stdout > tests/integration/codegen/polkadot.rs`
+/// - `cargo run --release -p subxt-cli -- codegen | rustfmt --edition=2018 --emit=stdout > subxt/tests/integration/codegen/polkadot.rs`
```

Comment:
We're on edition 2021 I think. But also: do we really need that? I suspect just `| rustfmt > subxt/…/…` works too?

Reply:
I agree; I don't think either of the flags are needed actually (I always just `rustfmt > outfile`)